schemas/common/dataImportEventTypes.json

```diff
@@ -14,6 +14,7 @@
     "DI_SRS_MARC_AUTHORITY_RECORD_CREATED",
     "DI_SRS_MARC_HOLDING_RECORD_CREATED",
     "DI_SRS_MARC_HOLDINGS_HOLDING_HRID_SET",
+    "DI_SRS_MARC_AUTHORITY_AUTHORITY_HRID_SET",
```

Comment:
We will not set hrid for authority records, so this event not needed.

Reply:
removed